python/lolopy/learners.py

```diff
@@ -316,36 +316,40 @@ class RandomForestClassifier(BaseLoloClassifier, RandomForestMixin):
 class RegressionTreeLearner(BaseLoloRegressor):
     """Regression tree learner, based on the RandomTree algorithm."""
 
-    def __init__(self, num_features=-1, max_depth=30, min_leaf_instances=1, leaf_learner=None,
-                 randomize_pivot_location=False):
```

Comment:
Note: we can still use randomize_pivot_locations in lolopy using the `RandomForestRegressor`.  This only removes control over the pivot locations when working with a single regression tree in lolopy (which I don't think is a super-high priority).  We could add this and general lolopy control over the splitters in another PR.